.github/workflows/post-merge-to-dev.yml

```diff
@@ -0,0 +1,30 @@
+name: DEV Post merge, deploy Docker build, ECR push, template copy to S3
+on:
+  push:
+    branches:
+      - main
+      - bau-add-deploy-to-dev
```

Comment:
Is this still necessary, I would nice have some context to the PR of why this necessary just for some guy in history wondering why this was done.